new-appx.ps1

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+if (Test-Path ./drop/productionEdge) { Remove-Item ./drop/productionEdge -Recurse }
+if (Test-Path ./drop/productionEdge.appx) { Remove-Item ./drop/productionEdge.appx }
+Copy-Item ./drop/production ./drop/productionEdge -Recurse
+Copy-Item ./AppXManifest.xml ./drop/productionEdge/AppXManifest.xml
+&"c:\Program Files (x86)\Windows Kits\10\bin\10.0.17134.0\x64\makeappx.exe" pack /h SHA256 /d .\drop\productionEdge /p .\drop\productionEdge.appx
```

Comment:
Is `10.0.17134.0` a version number (on the path we're using)
Seems a little brittle to have a "changing" version on the path.

Reply:
Fair question. 10.0.17134.0 is the Windows SDK version for the April 2018 update. The SDK install, at least on my machine, looks to be additive - old versions appear to be retained as new ones are added (my box has folders for the 5 most recent versions of Windows 10). So I don't expect this version to need to change during the time window that we'll be using this script.